src/guide/get-started/binaries.md

```diff
@@ -0,0 +1,143 @@
+```
+
+```shell [`kagami`]
+docker run -t hyperledger/iroha2:kagami-dev
+```
+
+## Using the Source GitHub Repository {#source}
+
+To perform any of the actions using the GitHub repository (i.e., [building & installing](#source-install), or [running](#source-run) binaries), the [Rust toolchain](https://www.rust-lang.org/) must first be installed. The toolchain is delivered with the package manager—[Cargo](https://doc.rust-lang.org/cargo/index.html)—that allows for installation of Rust crates from different sources, including GitHub.
+
+The recommended way to install the Rust toolchain is through the Rust installer and version management tool—[Rustup](https://rust-lang.github.io/rustup/) (See also: [Install Rust](https://www.rust-lang.org/tools/install))—by executing the following command in your terminal:
+
+  ```bash
+  curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+  ```
+
+::: info
+
+Depending on the operating system that you are using or other limitations, you may want to opt for a different installation method.
+
+For all available installation methods, see the [Rust Forge: Other Rust Installation Methods](https://forge.rust-lang.org/infra/other-installation-methods.html) article.
+
+In case you experience any issues with Rust installation, see our [Troubleshooting Rust Toolchain](/guide/troubleshooting/installation-issues#troubleshooting-rust-toolchain) topic.
+
+:::
+
+### Installing from Source GitHub Repository {#source-install}
+
+To perform a system-wide installation of an Iroha 2 binary with `cargo install`, execute one of the following commands in your terminal:
+
+::: code-group
+
+```shell [Command structure]
+cargo install [options] --git [repo url] --branch [branch name] [binary name]
+```
+
+```shell [`Iroha`]
+cargo install --git https://github.com/hyperledger/iroha.git --branch iroha2-dev iroha
+
+iroha --help
+```
+
+```shell [`Iroha Client CLI`]
+cargo install --git https://github.com/hyperledger/iroha.git --branch iroha2-dev iroha_client_cli
+
+iroha_client_cli --help
+```
+
+```shell [`Kagami`]
+cargo install --git https://github.com/hyperledger/iroha.git --branch iroha2-dev kagami
+
+kagami --help
+```
+
+:::
+
+::: tip
+
+For more details on `cargo install` and its `[options]`, see [The Cargo Book > cargo-install(1)](https://doc.rust-lang.org/cargo/commands/cargo-install.html).
+
+:::
+
+### Running from Cloned GitHub Repository {#source-run}
+
+First, clone the [hyperledger/iroha](https://github.com/hyperledger/iroha.git) GitHub repository:
+
+```shell
+git clone https://github.com/hyperledger/iroha.git <clone-folder>
+cd <clone-folder>
+git checkout iroha2-dev
+```
+
+To run an Iroha 2 binary with `cargo install`, execute one of the following commands in your terminal:
+
+::: code-group
+
+```shell [Command structure]
+cargo run [options] --bin [binary name] -- [arguments for the binary]
+```
+
+```shell [Iroha]
+cargo run --release --bin iroha -- --help
```

Comment:
```suggestion
cargo run --release --bin irohad -- --help
```

Reply:
Maybe add a comment that `--release` will result in better performance due to the project being built in the optimized release mode.